docs/natspec-format.rst

```diff
@@ -58,7 +58,7 @@ The following example shows a contract and a function using all available tags.
 
   This may change in the future.
 
-.. code:: Solidity
+.. code-block:: Solidity
```

Comment:
Does it matter if it's upper or lower case?

Reply:
I made the code work with both, so no.